sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/RecognizeContent.java

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer;
+
+import com.azure.ai.formrecognizer.models.FormContentType;
+import com.azure.ai.formrecognizer.models.FormPage;
+import com.azure.ai.formrecognizer.models.OperationResult;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.util.IterableStream;
+import com.azure.core.util.polling.SyncPoller;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+
+/**
+ * Sample for extracting layout information using input stream.
+ */
+public class RecognizeContent {
+
+    /**
+     * Main method to invoke this demo.
+     *
+     * @param args Unused. Arguments to the program.
+     *
+     * @throws IOException Exception thrown when there is an error in reading all the bytes from the File.
+     */
+    public static void main(final String[] args) throws IOException {
+        // Instantiate a client that will be used to call the service.
+
+        FormRecognizerClient client = new FormRecognizerClientBuilder()
+            .apiKey(new AzureKeyCredential("{api_key}"))
+            .endpoint("https://{endpoint}.cognitiveservices.azure.com/")
+            .buildClient();
+
+        File sourceFile = new File("../../test/resources/sample-files/layout1.jpg");
+        byte[] fileContent = Files.readAllBytes(sourceFile.toPath());
+        InputStream targetStream = new ByteArrayInputStream(fileContent);
+
+        SyncPoller<OperationResult, IterableStream<FormPage>> recognizeLayoutPoller =
+            client.beginRecognizeContent(targetStream, sourceFile.length(), FormContentType.IMAGE_JPEG);
```

Comment:
I thought java had automatic content type detection. If so, can you not pass the content type since that's what most users will be doing

Reply:
Considering the samples might get in before the feature would like to keep it this way and maybe consider updating once we get the feature in.